utils/src/check/super.h

```diff
@@ -4,7 +4,7 @@
 extern struct scoutfs_super_block *global_super;
 
 int check_super_crc(bool repair);
-int check_supers(int data_fd);
+int check_supers(int data_fd, bool repair);
```

Comment:
If you are adding a clobber that also can be repaired, you should modify `tests/test/clobber-repair` and include the clobber in the array `CLOBBERS` so that automated testing can be performed on the clobber and repair code.